pkg/virt-controller/services/multus_annotations.go

```diff
@@ -67,6 +69,18 @@ func GenerateMultusCNIAnnotationFromNameScheme(namespace string, interfaces []v1
 			multusNetworkAnnotationPool.add(
 				newMultusAnnotationData(namespace, interfaces, network, podInterfaceName))
 		}
+
+		if config != nil && config.NetworkBindingPlugingsEnabled() {
+			if iface := vmispec.LookupInterfaceByName(interfaces, network.Name); iface.Binding != nil {
+				podIfaceName := networkNameScheme[network.Name]
+				bindingPluginAnnotationData, err := newBindingPluginMultusAnnotationData(
+					config.GetConfig(), iface.Binding.Name, namespace, podIfaceName)
```

Comment:
You are still passing in the pod interface name, I do not understand why and how it is helpful for a general binding CNI. Why we are not passing the logical network name?

Reply:
Following offline discussion I changed it to the "logical" network name.